app/views/layouts/_facebook_script.html.erb

```diff
@@ -1,21 +1,75 @@
 <!-- Facebook script -->
-<div id="fb-root"></div>
 <script>
-window.fbAsyncInit = function() {
-  FB.init({
-    appId      : '<%= ENV["FACEBOOK_APP_ID"] %>',
-    xfbml      : true,
-    version    : 'v2.8'
+var bindFacebookEvents, defer, dispatch, handleEvent, initializeFacebookSDK;
+var loadFacebookSDK, loaded, restoreFacebookRoot, saveFacebookRoot;
+
+({defer, dispatch} = Turbolinks);
```

Comment:
I don't think this will work in many browsers...since you are using vanilla `var` to declare the variables, I think the intention is to use vanilla everywhere :)